tools/lisa-buildroot-update-kernel-config

```diff
@@ -53,6 +53,19 @@ fi
 
 set -u
 
+
+echo "Merging Kconfig fragment into $KERNEL_CONFIG"
+fragment=$(mktemp)
+
 # Update kernel .config to use the provided rootfs as builtin initramfs
-sed -i "s#[.]*CONFIG_INITRAMFS_SOURCE.*#CONFIG_INITRAMFS_SOURCE=\"$ROOTFS\"#" "$KERNEL_CONFIG"
-make -C "$KERNEL_DIR" olddefconfig
+# Setting root UID and GID also seems to be necessary.
```

Comment:
Can you expand on this comment please? Why is it necessary, what breaks if this is not set?

Reply:
`Kconfig` asks you in an interactive way what you want, in a similar way as `make oldconfig`. Not sure why exactly, since we just need the default value though. I haven't investigated deeply since I could not find any information on the exact expected behavior of that Kconfig fragment script.

Reply:
`make olddefconfig` doens't ask interactive questions. It keeps the current configs but updates it to be correct in case you modified something and there are some dependencies on it like in your case. `make olddefconfig` is the right method to deal with these cases.

Reply:
Sorry, I meant running `make Image`, the issue does not trigger when running `make oldconfig` but at the next step.